README.md

```diff
@@ -58,28 +78,31 @@ pnpm install
 pnpm dev
 ```
 
-4. **Docker Compose**
+### 5. Accessing the Services
 
-Launch the services using Docker Compose:
+- **server:** Exposes the management application APIs, available on port 4000 by default.
+- **web:** Exposes the management application interface, available on port 3000 by default.
+- **kong:** Exposes the API gateway APIs, available on ports 8000 (development) and 8100 (production) by default.
+- **logstash:** Collects logs from the Kong service.
+- **elasticsearch:** Provides search and analytics on logs from Kong.
+- **maildev (`dev` only):** A developer interface for testing emails. The web app runs on [http://localhost:1080]().
```

Comment:
Can we not use the `dev` profile for this? Happy to move it, just highlighting that if this goes against the intention of the profiles, I'm not sure how they're used.

Reply:
the dev profile in this case is for running dev environment services, which would be exposed to ACs to test out the platform/APIs, valid EMAIL_ variables would need to be used for this, the issue with maildev here is that its simply used for local development/testing purposes, so dont think it should be bundled as part of the gateways services.

Reply:
Thanks for the clarification. I'll decouple them.

Reply:
I just revisited this, but have concluded my only real options are:

1. make a change that breaks some type safety in the application by allowing `EMAIL_HOST` to be a non-domain string, which would devs to connect to the maildev container using `localhost`
2. expose `obn-net` so that `server` can connect to maildev, which also seems kind of bad
3. use the `dev` profile

Let me know what you think. I might be missing some other way to include it. I can remove it, but I think it's nice for development/testing.

Reply:
lets go with 2, alongside this https://github.com/openbankingnigeria/obn-gateway/pull/203#discussion_r1605596633

it is fine to have the maildev container stay on the same `obn-net` network as the server container, its pretty much the same behavior as you've included on the docker compose file.

pretty much asking for it to be removed from the compose file to avoid bloat, as the maildev service isnt critical to the open banking project, and maildev is supposed to be simply a helper tool for those who dont have their own mail servers, which should ideally be independent of what we're building.